src/pages/Autolending/GenderRadios.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<div class="gender-radios">
```

Comment:
I think it might make sense to include the title in this component too `<h3>Gender</h3>`

Reply:
💯